registry/handlers/context.go

```diff
@@ -45,8 +45,8 @@ func getName(ctx context.Context) (name string) {
 	return ctxu.GetStringValue(ctx, "vars.name")
 }
 
-func getTag(ctx context.Context) (tag string) {
-	return ctxu.GetStringValue(ctx, "vars.tag")
+func getReference(ctx context.Context) (reference string) {
+	return ctxu.GetStringValue(ctx, "vars.reference")
```

Comment:
Need to add an entry for vars.reference to https://github.com/docker/distribution/blob/master/registry/handlers/app.go#L278-L283